### PR TITLE
WebGL: use color grading in demo, expose destroy.

### PR DESCRIPTION
**Pull request by prideout**

### CODE REVIEW

File: web/samples/helmet.html [hunk 1]
```diff
@@ -57,7 +57,7 @@
         const sunlight = Filament.EntityManager.get().create();
         Filament.LightManager.Builder(LightType.SUN)
             .color(iblColor.slice(0, 3))
-            .intensity(150000)
```
Comment (romainguy) [inline web/samples/helmet.html:60]: That's quite a change in intensity Oo

File: web/samples/helmet.html [hunk 2]
```diff
@@ -95,13 +95,17 @@
 
         const cameraEntity = Filament.EntityManager.get().create();
         this.camera = engine.createCamera(cameraEntity);
-        this.camera.setExposure(32, 1/125.0, 100);
+
+        const colorGrading = Filament.ColorGrading.Builder()
+            .toneMapping(Filament.ColorGrading$ToneMapping.ACES)
```
Comment (romainguy) [inline web/samples/helmet.html:100]: You could add a vignette if you want to make it obvious

File: web/samples/helmet.html (hunk 1 above)
Comment (prideout) [reply web/samples/helmet.html:60]: Fixed.

File: web/samples/helmet.html (hunk 2 above)
Comment (prideout) [reply web/samples/helmet.html:100]: Oh, actually I need to add bindings for that, it's a View API not a ColorGrading API.  Will do so next.

**Pull request merged**